package.json

```diff
@@ -1,7 +1,9 @@
 {
   "name": "openstreetmap",
   "private": true,
-  "dependencies": {},
+  "dependencies": {
+    "bs-custom-file-input": "^1.3.4"
+  },
```

Comment:
We haven't been using yarn to manage javascript for the actual web site to date, although I have experimented with it. I was slightly surprised it worked but I see `node_modules` has crept on to the include path at some point, and the other infrastructure change I made for it has also been made for other reasons.

I guess there's no reason not to do it, other than trying to keep everything in one place - although is other stuff we could manage via yarn there are some things where we can't at the moment so would have to stick with vendorer.

It does mean that we will need to run yarn as part of the deployment though and to date we don't have any tooling for that.